packages/rum-core/src/domain/configuration.ts

```diff
@@ -129,6 +133,10 @@ export function validateAndBuildRumConfiguration(
       defaultPrivacyLevel: objectHasValue(DefaultPrivacyLevel, initConfiguration.defaultPrivacyLevel)
         ? initConfiguration.defaultPrivacyLevel
         : DefaultPrivacyLevel.MASK,
+      enablePrivacyForActionName:
+        (isExperimentalFeatureEnabled(ExperimentalFeature.ENABLE_PRIVACY_FOR_ACTION_NAME) &&
+          initConfiguration.enablePrivacyForActionName) ??
+        false,
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏ ‏To ensure we sanitise as boolean you should use: 
```suggestion
      enablePrivacyForActionName:
        isExperimentalFeatureEnabled(ExperimentalFeature.ENABLE_PRIVACY_FOR_ACTION_NAME) &&
          !!initConfiguration.enablePrivacyForActionName
```